activemodel/lib/active_model/errors.rb

```diff
@@ -407,6 +390,8 @@ def added?(attribute, message = :invalid, options = {})
     def full_messages
       map { |attribute, message| full_message(attribute, message) }
     end
+    # aliases full_messages
```

Comment:
That's true. I wanted to be consistent with other comments like https://github.com/morgoth/rails/blob/activemodel-errors-refactoring/activemodel/lib/active_model/errors.rb#L105-L108 but I can remove them all